data/actions/scripts/other/doors.lua

```diff
@@ -1,3 +1,51 @@
+local positions = {
+	{x = 1, y = 0}, -- east
+	{x = -1, y = 1}, -- south
```

Comment:
It would be cleaner if you just use the actual offsets here e.g. `Position(1, 0, 0) -- east`, `Position(0, 1, 0) -- south` etc.